packages/__docs__/src/Header/props.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2015 - present Instructure, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+import PropTypes from 'prop-types'
+import type { PropValidators } from '@instructure/shared-types'
+type HeaderOwnProps = {
+  name: string
+  version: string
+  versionsData: {
+    latestVersion: string
+    previousVersions: string[]
+  }
+}
+
+type PropKeys = keyof HeaderOwnProps
+
+type AllowedPropKeys = Readonly<Array<PropKeys>>
+
+type HeaderProps = HeaderOwnProps
+
+const propTypes: PropValidators<PropKeys> = {
+  name: PropTypes.string.isRequired,
+  version: PropTypes.string.isRequired,
+  versionsData: PropTypes.shape({
+    latestVersion: PropTypes.string.isRequired,
+    previousVersions: PropTypes.arrayOf(PropTypes.string).isRequired
+  })
+}
+
+const allowedProps: AllowedPropKeys = ['name', 'version', 'versionsData']
```

Comment:
You defined the `allowedProps` here, but forgot to add it to the component in the `index.tsx` :) Also the same problem with ComponentTheme, ContentWrap, Document, Glyph, Guidelines, Header, Hero and Icon components.